timer@Severga/README.md

```diff
@@ -0,0 +1,11 @@
+This is a timer Cinnamon applet.
+
+## Requirements
+This applet requires (optional):
+
+1) sox package (to play sounds);
+2) zenity package (to show system notifications).
```

Comment:
You can do perform both playing sounds as well as sending notifications with Cinnamon, and avoid spawning commands - 

for sounds, check out:
https://github.com/linuxmint/Cinnamon/blob/master/js/ui/soundManager.js

For notifications, have a look at this example:
https://github.com/linuxmint/Cinnamon/blob/master/files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js#L838-L845

Reply:
Thanks again! I will check these out and consider. To tell the truth, I tried to use a part of other applet's code to send notifications but it wasn't working for me, and I decided to use "zenity".

Reply:
Unfortunately, I cannot make my script play sounds with the internal means of Cinnamon. The `global.display` object in my system doesn't contain `get_sound_player` function. There is `global.play_sound_file` function. But I also can not make it play anything. :(